utils/R/read.output.R

```diff
@@ -123,8 +123,6 @@ read.output <- function(runid, outdir, start.year=NA,
 
   if(!nofiles){
     for(ncfile in ncfiles) {
-      valid.nc <- tryl(nc_open(ncfile))
```

Comment:
what shall be the condition to check in this case? Something like `class(valid.nc)=="ncdf4"` (or `class(valid.nc)!="try-error")?` 
```
 valid.nc <- try(nc_open(ncfile))
      if(class(valid.nc)=="ncdf4"){...}
```